src/library/trackmodel.h

```diff
@@ -49,6 +49,7 @@ class TrackModel {
     Q_DECLARE_FLAGS(Capabilities, Capability)
 
     enum SortColumnId {
+        SORTCOLUMN_CURRENTINDEX = -2, // Column with the cursor on it
```

Comment:
Can you swap these two, so that current is -1?i think we can safely do that.

Reply:
I swapped the numerical values!